exercises/practice/protein-translation/.meta/example.rkt

```diff
@@ -0,0 +1,24 @@
+#lang racket
+
+(provide proteins)
+
+(define (translate protein)
+  (case protein
+    [("AUG") "Methionine"]
+    [("UUU" "UUC") "Phenylalanine"]
+    [("UUA" "UUG") "Leucine"]
+    [("UCU" "UCC" "UCA" "UCG") "Serine"]
+    [("UAU" "UAC") "Tyrosine"]
+    [("UGU" "UGC") "Cysteine"]
+    [("UGG") "Tryptophan"]
+    [("UAA" "UAG" "UGA") '()]
```

Comment:
This line should return a empty string for consistency with the other branches so there aren't any rude surprises. Racket has an `non-empty-string` function you can negate to check if the return value is in fact an empty string.